schema_based_on_diagram.sql

```diff
@@ -0,0 +1,118 @@
+CREATE TABLE patients (
+    id PRIMARY KEY,
+    name VARCHAR(100),
+    date_of_birth DATE,
+)
+
+CREATE TABLE medical_histories(
+    id PRIMARY KEY
+    admitted at TIMESTAMP,
+    patient_id REFERENCES patients(id),
+    status VARCHAR(50)
+)
+--table for many-to-many relations
+CREATE TABLE medicals(
+    medical_history_id REFERENCES medical_histories(id),
+    treamtment_id REFERENCES treamtments(id)
+)
+CREATE TABLE treatmets(
+    id PRIMARY KEY,
+    type VARCHAR,
+    name VARCHAR
+)
+
+CREATE TABLE patients (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(100),
+    date_of_birth DATE
+);
+
+CREATE TABLE medical_histories (
+    id SERIAL PRIMARY KEY,
+    admitted_at TIMESTAMP,
+    patient_id INTEGER REFERENCES patients(id),
+    status VARCHAR(50)
+);
+
+-- Table for many-to-many relations
+CREATE TABLE medicals (
+    medical_history_id INTEGER REFERENCES medical_histories(id),
+    treatment_id INTEGER REFERENCES treatments(id)
+);
+
+CREATE TABLE treatments (
+    id SERIAL PRIMARY KEY,
+    type VARCHAR(50),
+    name VARCHAR(100)
+);
```

Comment:
- [ ] Great job so far @husush23 and @Katended There are just a few changes I will suggest you implement and you'll be on your way to the next milestone :wink: 
- [ ] I noticed that these tables have been duplicated so kindly remove the duplicates so your code only contains one each of thes tables :+1: